context/brand-context/nature/scss/10-settings/typography.scss

```diff
@@ -8,37 +8,37 @@ $context--font-family-serif: if($is-nature-branded, (Harding, Palatino, serif),
 $context--font-family-primary: if($is-nature-branded, $context--font-family-serif, $context--font-family-sans);
 $context--font-family-secondary: $context--font-family-sans;
 
-$context--font-size-xxs: 12px;
-$context--font-size-xs: 14px;
-$context--font-size-sm: 16px;
-$context--font-size-md: 18px;
-$context--font-size-lg: 24px;
-$context--font-size-xl: 28px;
-$context--font-size-xxl: 32px;
-
-$context--font-size-h1: 3.2rem;
-$context--font-size-h2: 2.4rem;
-$context--font-size-h3: 2.0rem;
-$context--font-size-h4: 1.8rem;
-
-$context--line-height-h1: 3.6rem;
-$context--line-height-h2: 2.6rem;
-$context--line-height-h3: 2.2rem;
-$context--line-height-h4: 2.2rem;
+$context--font-size-xxs: .75rem;
+$context--font-size-xs: .875rem;
+$context--font-size-sm: 1rem;
+$context--font-size-md: 1.125rem;
+$context--font-size-lg: 1.5rem;
+$context--font-size-xl: 1.75rem;
+$context--font-size-xxl: 2rem;
+
+$context--font-size-h1: 2rem;
+$context--font-size-h2: 1.5rem;
+$context--font-size-h3: 1.25rem;
+$context--font-size-h4: 1.25rem;
+
+$context--line-height-h1: 2.25rem;
+$context--line-height-h2: 1.6rem;
+$context--line-height-h3: 1.4rem;
+$context--line-height-h4: 1.4rem;
 
 // Fluid typography
 $context--font-size-h1-min: $context--font-size-h2;
 $context--font-size-h2-min: $context--font-size-h3;
 $context--font-size-h3-min: $context--font-size-h4;
-$context--font-size-h4-min: 1.6rem; // Body typography size
+$context--font-size-h4-min: 1rem; // Body typography size
 
 $context--line-height-h1-min: $context--line-height-h2;
 $context--line-height-h2-min: $context--line-height-h3;
 $context--line-height-h3-min: $context--line-height-h4;
 $context--line-height-h4-min: $context--line-height-h4;
 
-$context--letter-spacing-min: -0.01875rem;
-$context--letter-spacing: -0.0625rem;
+$context--letter-spacing-min: -0.01875rem; // TODO - What is this?
```

Comment:
These variables are used by https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/nature/scss/30-mixins/typography.scss for setting letter-spacing on heading elements. We should probably update the values to be relative to 100% on the html.

Reply:
> We should probably update the values to be relative to 100% on the html.

I can do this as part of the other changes - so hopefully, visually, _nothing_ changes<sup>*<sup>

<sup>*</sup> famous last words.

Reply:
Oh are those changes happening in a different PR?

I think it's just a case of updating to:
```
$context--letter-spacing-min: 0.001171875rem;
$context--letter-spacing: -0.0390625rem;
```

Reply:
no, they're as part of this PR - I need to change the title a little I guess and add more detail. 

I've gone through and changed any css rules that effect type that is set in `px` or `rem` to match a root font-size of `100%`.
If there are rules set in `em` I have left them as they should respect the body font size that is `18px` (or `1.125rem` now (it was `1.8rem`).

So here, because they're set in `rem` they need changing. 

I got

```
$context--letter-spacing-min: -0.011715625rem; // in Safari this is 0.18745000660419464px not 0.1875px
$context--letter-spacing: -0.0390625rem;
```

So will double check and update accordingly.